Core/GDCore/Vector2.h

```diff
@@ -0,0 +1,360 @@
+// This code comes from the SFML.
```

Comment:
Should it be formatted?

Reply:
Not mandatory for C++ files (for now).

```suggestion
// This is adapted from SFML (https://github.com/SFML/SFML).
```